lib/Cake/Console/Command/Task/ControllerTask.php

```diff
@@ -328,7 +328,7 @@ public function bakeActions($controllerName, $admin = null, $wannaUseSession = t
  * @param array $components Components to use in controller
  * @return string Baked controller
  */
-	public function bake($controllerName, $actions = '', $helpers = null, $components = null) {
+	public function bake($controllerName, $actions = '', $helpers = array(), $components = array()) {
```

Comment:
Why did you change this signature?

Reply:
It is a change to correct errors that occur in the following lines.

https://github.com/cakephp/cakephp/blob/2.x/lib/Cake/Console/Templates/default/classes/controller.ctp#L51

But it is not BC with existing applications.
Make `null` be converted to `array()`.